ui/src/App.tsx

```diff
@@ -68,43 +73,48 @@ const App = () => {
           style={{ height: 500 }}
         />
       </div>
-      <label>Title</label>
+      <label htmlFor="title">Title</label>
       <input
+        id="title"
         type="text"
         onChange={(e) => {
           setTitle(e.target.value);
         }}
         value={title}
       />
 
-      <label>Start Date</label>
+      <label htmlFor="startDate">Start Date</label>
       <input
+        id="startDate"
         min={formatDate(new Date())}
         type="date"
         onChange={(e) => {
           setStartDate(e.target.value);
         }}
         value={startDate}
       />
-      <label>Start Time</label>
+      <label htmlFor="startTime">Start Time</label>
       <input
+        id="startTime"
         type="time"
         onChange={(e) => {
           setStartTime(e.target.value);
         }}
         value={startTime}
       />
-      <label>End Date</label>
+      <label htmlFor="endDate">End Date</label>
       <input
+        id="endDate"
         min={startDate}
         type="date"
         onChange={(e) => {
           setEndDate(e.target.value);
         }}
         value={endDate}
       />
-      <label>End Time</label>
+      <label htmlFor="endTime">End Time</label>
```

Comment:
What does this `htmlFor` do? Why do we need it here?

Reply:
We wanted to tie the label to its corresponding input field so we could find the input via its label and preform an action on it. If you look at the input below this label you will see it has an `id` attribute of "endTime" as well which connects it to the label above. This helped out in our tests to be able to retrieve the input by it's label but is also considered a good practice for making an accessibility friendly application since screen readers use the label text to read out to the user what kind of input they're hovering over.

Here's some cool articles on more about creating a relationship between an input field and it's label if you're interested in learning more:
- https://css-tricks.com/html-inputs-and-labels-a-love-story/
- https://www.coolfields.co.uk/2011/04/accessible-forms-should-every-input-have-a-label/#:~:text=Firstly%2C%20labels%20are%20an%20accessibility,is%20required%20from%20the%20user.

Reply:
Cool, good to know :)